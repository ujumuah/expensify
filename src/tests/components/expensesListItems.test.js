import React from 'react'
import {shallow} from 'enzyme'
import Items from '../../components/pages/expensesListItems'
import  expensesSample from  '../expenses/samples'
test('should render Items with expensesSaple data',()=>{
    const wrapper = shallow(<Items {...expensesSample[0]}/>)
    expect(wrapper).toMatchSnapshot()
})