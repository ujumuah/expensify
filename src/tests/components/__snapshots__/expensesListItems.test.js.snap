// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Items with expensesSaple data 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h4>
      Expense:
      rent
    </h4>
  </Link>
  <div>
    Amount:
    1225
  </div>
  <div>
    Created on :
    01-Jan-1970
  </div>
</div>
`;
